tests/bettertransformer/test_bettertransformer_vision.py

```diff
@@ -133,3 +223,94 @@ def test_raise_autocast(self, test_name: str, model_id, padding, max_length=20):
     )
     def test_raise_train(self, test_name: str, model_id, padding, max_length=20):
         super().test_raise_train([model_id], padding=padding, max_length=max_length)
+
+    @parameterized.expand(
+        grid_parameters(
+            {
+                "model_id": all_models_to_test,
+                "keep_original_model": [True, False],
+            }
+        )
+    )
+    def test_invert_modules(self, test_name: str, model_id, keep_original_model=False):
+        super().test_invert_modules(model_id=model_id, keep_original_model=keep_original_model)
+
+    @parameterized.expand(
+        grid_parameters(
+            {
+                "model_id": all_models_to_test,
+                "keep_original_model": [True, False],
+            }
+        )
+    )
+    def test_save_load_invertible(self, test_name: str, model_id, keep_original_model=False):
+        super().test_save_load_invertible(model_id=model_id, keep_original_model=keep_original_model)
+
+    @parameterized.expand(
+        grid_parameters(
+            {
+                "model_id": all_models_to_test,
+                "keep_original_model": [True, False],
+                "padding": ["max_length", True],
+            }
+        )
+    )
+    def test_invert_model_logits(self, test_name: str, model_id, keep_original_model=False, padding=True):
+        r"""
+        Test that the inverse converted model and hf model have the same logits
+        """
+        # get hf and bt model
+        hf_model = AutoModel.from_pretrained(model_id)
+        # get bt model and invert it
+        bt_model = BetterTransformer.transform(hf_model, keep_original_model=keep_original_model)
+        bt_model = BetterTransformer.reverse(bt_model)
+
+        # get inputs
+        inputs = self.prepare_inputs_for_class(model_id, padding=padding)
+
+        # get outputs
+        torch.manual_seed(42)
+        output_bt = bt_model(**inputs)
+
+        torch.manual_seed(42)
+        output_hf = hf_model(**inputs)
+
+        # Assert that the outputs are the same
+        self.assertTrue(torch.allclose(output_bt[0], output_hf[0], atol=1e-3))
```

Comment:
This test is always passing in case `keep_original_model=False`.

Reply:
Thanks for spotting ! 
Fixed in [43fb900](https://github.com/huggingface/optimum/pull/676/commits/43fb9009a66de6de1765ec684ce53395464f9c43)